_data/projects/Hostel-Pass-Management-Application.yml

```diff
@@ -0,0 +1,14 @@
+name: Hostel-Pass-Management-Application
+desc: A platform independent application written in Java for management of Hostel Pass for students.
+site: https://github.com/NishkarshRaj/Hostel-Pass-Management-Application
+tags:
+- java
+- devops
+- management
+- collaboration
+- maven
+- jenkins
+- automation
+upforgrabs:
+  name: Up-for-Grabs
```

Comment:
The `upforgrabs.name` value 'Up-for-Grabs' isn't in use on the project in GitHub. This might just be a mistake due because of copy-pasting the reference template or be mis-typed. Please check the list of labels at https://github.com/NishkarshRaj/Hostel-Pass-Management-Application/labels and update the project file to use the correct label.

Reply:
Yes sir,
I have updated the label from "Up for Grabs" to "Up-for-Grabs" in the project on GitHub.
Please review.

Reply:
👍 might want to clean up the previous label:

<img width="224" src="https://user-images.githubusercontent.com/359239/69252014-b8140900-0b88-11ea-85aa-a1b2e83edc18.png">

Reply:
I have deleted the previous label now 👍